iosApp/iosApp/LaunchView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  LaunchView.swift
+//  iosApp
+//
+//  Created by Nasir Ahmed Momin on 20/05/22.
+//  Copyright © 2022 Mutual Mobile. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+struct LaunchView: View {
+    
+    @State private var selection = 0
+    
+    private var textList = [
+        "Simple, powerful time tracking, reporting and invoicing",
```

Comment:
these should come from localise file

Reply:
mark this as resolved if we are not following localization for now.
But if we are following, then all text needs to be localised

Reply:
Thanks for review, Added a localisation file & moved all string in that file,

Changes are available in next MR